qiskit_ibm_runtime/accounts/account.py

```diff
@@ -235,47 +222,6 @@ def _assert_valid_instance(instance: str) -> None:
         pass
 
 
-class QuantumAccount(Account):
```

Comment:
Luckily this class is not part of the documented interface, so no deprecation is needed before the removal.